src/Raven.Server/Documents/ChangesClientConnection.cs

```diff
@@ -536,13 +536,13 @@ public void Confirm(int commandId)
             });
         }
 
-        public void EnableTopologyChangeNotification()
+        public void SendSupportedFeatures()
         {
             _sendQueue.Enqueue(new ChangeValue
             {
                 ValueToSend = new DynamicJsonValue
                 {
-                    ["EnableTopologyChangeNotification"] = ""
+                    ["TopologyChange"] = "true"
```

Comment:
just `true` not `"true"`

then you can do `json.TryGet(nameof(TopologyChange), out bool supports)`